engine/src/main/assets/rpgmaker/rpgmv-v2-joi/shims/node_modules__buffer__index.js

```
@@ -0,0 +1,2108 @@
+
+// ArrayBuffer or Uint8Array objects from other contexts (i.e. iframes) do not pass
+// the `instanceof` check but they should be treated as of that type.
+// See: https://github.com/feross/buffer/issues/166
+function isInstance (obj, type) {
+  return obj instanceof type ||
+    (obj != null && obj.constructor != null && obj.constructor.name != null &&
+      obj.constructor.name === type.name)
+}
+function numberIsNaN (obj) {
+  // For IE11 support
+  return obj !== obj // eslint-disable-line no-self-compare
```

Biome (2.5.8)

[error] This comparison uses the same expression on both sides.

(lint/suspicious/noSelfCompare)